### DISCUSSION TITLE
how to get query result ts

### DISCUSSION THREAD

**Discussion by zhihengGet** [Automattic/mongoose#10836] [General]
i got following error when trying to access n nmodified  ok ... 

`Property "n"   "nModifed"   does not exist on type 'UpdateWriteOpResult'.ts(2339)`

and are we going to migrate to deletedCount, matchedCount, upsertedCount ....etc  from n and nModified  ?  i remember there was a time query returned matchedCount instead of n but now it is back to n again ...bc i changed mongoose version to latest ?_ 

```
let res = await TestModel.updateMany({}, { someProperty: 'someValue' });

res.matchedCount; // Number of documents that were found that match the filter. Replaces `res.n`
res.modifiedCount; // Number of documents modified. Replaces `res.nModified`
res.upsertedCount; // Number of documents upserted. Replaces `res.upserted`

```
or

 `res.n`
`res.nModified`
`res.upserted`

_my query gives me n and nModifed_ 

**mongodb: 4.1**

**mongoose: lastest**

**Comment by vkarpov15** [comment]
Mongoose 6 does not have `n` or `nModified` anymore. Here's the relevant docs in the migration guide: https://mongoosejs.com/docs/migrating_to_6.html#mongodb-driver-40
